app/forms/sufia/forms/work_form.rb

```diff
@@ -7,5 +7,17 @@ def rendered_terms
                :visibility_after_embargo, :visibility_during_lease,
                :lease_expiration_date, :visibility_after_lease, :visibility, :thumbnail_id, :representative_id, :ordered_member_ids]
     end
+
+    # override Curation Concerns Work Form, Sufia allows multiple values for rights
+    def self.multiple?(term)
+      case term.to_s
+      when 'rights'
+        true
+      when 'ordered_member_ids'
```

Comment:
@mjgiarlo: I think this was about the comment line I put previously, which is removed now, but I might be wrong.